### PR TITLE
Corrected  the flow and grammar in code_of_conduct.md file

### PR DESCRIPTION
**Pull request by Vanshiii203**
- **What kind of change does this PR introduce?** 
This PR updates the docs to make text flow better.

- **Why was this change needed?** 
issue #1897

### CODE REVIEW

**Comment by Vanshiii203** [comment]
@p-fernandez Please review and merge this

**Comment by Vanshiii203** [comment]
@ainouzgali Can you please review and merge this?

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -2,50 +2,45 @@
...
 
-We as members, contributors, and leaders pledge to make participation in our
-community a harassment-free experience for everyone, regardless of age, body
-size, visible or invisible disability, ethnicity, sex characteristics, gender
-identity and expression, level of experience, education, socio-economic status,
-nationality, personal appearance, race, religion, or sexual identity
-and orientation.
+As members, contributors, and leaders, we pledge to ensure that everyone can participate 
+in our community without harassment regardless of age, body size, visible or invisible disability,
+ethnicity, sex characteristics, gender identity, or level of experience.
```
Comment (p-fernandez) [inline CODE_OF_CONDUCT.md:7]: You removed education, socio-economic status, nationality, race, religion and sexual orientation. Please add them back.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -2,50 +2,45 @@
...
+
 
 We pledge to act and interact in ways that contribute to an open, welcoming,
-diverse, inclusive, and healthy community.
+diverse, inclusive, diverse and healthy community.
```
Comment (p-fernandez) [inline CODE_OF_CONDUCT.md:11]: This adds repetition and it is not needed.

File: CODE_OF_CONDUCT.md [hunk 3]
```diff
@@ -2,50 +2,45 @@
...
 * Accepting responsibility and apologizing to those affected by our mistakes,
-  and learning from the experience
-* Focusing on what is best not just for us as individuals, but for the
-  overall community
+  and learning from the experience.
+* Focusing on what is best not just for us as individuals but for the
+  overall community.
 
-Examples of unacceptable behavior include:
+Examples of unacceptable behaviour include:
```
Comment (p-fernandez) [inline CODE_OF_CONDUCT.md:26]: There is not an agreement in the core maintainers of using british english. Most of the codebase and documentation is written in american english so this change it feels opinionated.

File: CODE_OF_CONDUCT.md [hunk 4]
```diff
@@ -2,50 +2,45 @@
...
 
-* The use of sexualized language or imagery, and sexual attention or
-  advances of any kind
-* Trolling, insulting or derogatory comments, and personal or political attacks
-* Public or private harassment
+* The use of sexualized language or imagery, as well as any form of sexual attention or advance.
```
Comment (p-fernandez) [inline CODE_OF_CONDUCT.md:28]:
```suggestion
* The use of sexualized language or imagery, as well as any form of sexual attention or advances of any kind.
```

File: CODE_OF_CONDUCT.md [hunk 5]
```diff
@@ -2,50 +2,45 @@
...
+* Trolling, insulting or derogatory comments, and personal or political attacks.
+* Public or private harassment.
 * Publishing others' private information, such as a physical or email
-  address, without their explicit permission
-* Other conduct which could reasonably be considered inappropriate in a
-  professional setting
+  address, without their explicit permission.
+* * Any other behaviour that could be considered inappropriate in a professional setting.
```
Comment (p-fernandez) [inline CODE_OF_CONDUCT.md:33]: Double asterisk.

File: CODE_OF_CONDUCT.md [hunk 6]
```diff
@@ -2,50 +2,45 @@
...
 Community leaders are responsible for clarifying and enforcing our standards of
 acceptable behavior and will take appropriate and fair corrective action in
-response to any behavior that they deem inappropriate, threatening, offensive,
-or harmful.
+the event of inappropriate, threatening, offensive, or harmful behaviour.
 
 Community leaders have the right and responsibility to remove, edit, or reject
 comments, commits, code, wiki edits, issues, and other contributions that are
-not aligned to this Code of Conduct, and will communicate reasons for moderation
+contrary to this Code of Conduct, and will communicate reasons for moderation
```
Comment (p-fernandez) [inline CODE_OF_CONDUCT.md:43]: Not aligned is a better legal term in this case.

File: CODE_OF_CONDUCT.md [hunk 7]
```diff
@@ -58,13 +53,12 @@ representative at an online or offline event.
...
 
-Instances of abusive, harassing, or otherwise unacceptable behavior may be
+Instances of abusive, harassing, or otherwise unacceptable behaviour may be
 reported to the community leaders responsible for enforcement at
 support@novu.co.
 All complaints will be reviewed and investigated promptly and fairly.
 
-All community leaders are obligated to respect the privacy and security of the
-reporter of any incident.
+All community leaders are obligated to respect the privacy and security of witnesses to any incident.
```
Comment (p-fernandez) [inline CODE_OF_CONDUCT.md:61]: Roll back to previous copy as this one changes the original meaning intended.

File: CODE_OF_CONDUCT.md [hunk 8]
```diff
@@ -73,17 +67,15 @@ the consequences for any action they deem in violation of this Code of Conduct:
...
 ### 1. Correction
 
-**Community Impact**: Use of inappropriate language or other behavior deemed
-unprofessional or unwelcome in the community.
+**Community Impact**: Use of inappropriate language or other unprofessional or unwelcome behaviour in the community.
 
 **Consequence**: A private, written warning from community leaders, providing
 clarity around the nature of the violation and an explanation of why the
-behavior was inappropriate. A public apology may be requested.
+behaviour was inappropriate. A public apology may be requested.
```
Comment (p-fernandez) [inline CODE_OF_CONDUCT.md:74]: Already flagged it before. Opinionated.

File: CODE_OF_CONDUCT.md [hunk 9]
```diff
@@ -105,9 +97,7 @@ Violating these terms may lead to a permanent ban.
 
 ### 4. Permanent Ban
 
-**Community Impact**: Demonstrating a pattern of violation of community
-standards, including sustained inappropriate behavior,  harassment of an
-individual, or aggression toward or disparagement of classes of individuals.
+**Community Impact**: Demonstrating a pattern of violating community standards, such as persistent inappropriate behavior, harassment of individuals, or aggression toward or disparagement of groups of people.
```
Comment (p-fernandez) [inline CODE_OF_CONDUCT.md:100]: I consider sustained is more precise in the goal of this point as it gives the meaning of a lesser frequency than persistent.

**Review by p-fernandez — CHANGES_REQUESTED** [review]

**Comment by scopsy** [comment]
Hi @Vanshiii203 did you had a chance to take a look at @p-fernandez comments? Let me know if you will have the time for that.

**Comment by scopsy** [comment]
Closing this for now due to inactivity. 🙏

**Pull request closed without merging**